src/ChannelServer/World/MonsterBuilder.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Melia.Shared.Const;
+using Melia.Shared.World;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Melia.Channel.World
+{
+    class MonsterBuilder
```

Comment:
What are you trying to achieve with this class? Why not simply use the Monster class?